src/app/archivedRows/ToggleArchivedRowsButton.jsx

```diff
@@ -1,31 +1,98 @@
-import React, { useCallback, useState } from "react";
+import { t } from "i18next";
+import React, { useEffect, useRef, useState } from "react";
 import { useDispatch, useSelector } from "react-redux";
+import SvgIcon from "../components/helperComponents/SvgIcon";
+import { outsideClickEffect } from "../helpers/useOutsideClick";
+import actionCreators from "../redux/actionCreators";
 import Action from "../redux/actionCreators";
 import { selectShowArchivedState } from "../redux/reducers/tableView";
+import { ShowArchived } from "./helpers";
 
-const ToggleArchivedRowsButton = ({ table }) => {
+const StateCfg = {
+  [ShowArchived.hide]: {
+    icon: <SvgIcon icon="/img/icons/database-current.svg" />,
+    trnKey: "only-unarchived"
+  },
+  [ShowArchived.show]: {
+    icon: <SvgIcon icon="/img/icons/database-current-archived.svg" />,
+    trnKey: "all"
+  },
+  [ShowArchived.exclusive]: {
+    icon: <SvgIcon icon="/img/icons/database-archived.svg" />,
+    trnKey: "only-archived"
+  }
+};
```

Comment:
Fand es jetzt erst bissl verwirrend, wie man vom archived state im button auf den `trnKey` zu kommen. 
Wenn man drüber nachdenkt, versteht man es schon,  ich frag mich aber ob es nicht einfacher gegangen wäre, wenn man direkt den trnKey als state in den buttons verwendet hätte?
Jetzt ist es sauber getrennt, aber man muss die Transferleistung halt einmal machen. 
Für mich passts so auch, ich wollts mir nur von der Seele schreiben

Reply:
Was heißt denn `trnKey` überhaupt?

Reply:
Das ist der finale Suffix des Keys, der in den i18n-JSON-Files zum String-Lookup genutzt wird. Translation Key oder so.